### PR TITLE
Changelog update for 202012.00

### PR DESCRIPTION
**Pull request by pavanmr94**
Description
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] My code is Linted.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,85 @@
...
+- The AWS IoT Jobs library has no dependencies on additional libraries other than the standard C library. It also doesn’t have any platform dependencies, such as threading or synchronization. It can be used with any MQTT library and any JSON library (see demos (https://github.com/aws/amazon-freertos/tree/master/demos/jobs_for_aws) with coreMQTT and coreJSON).
+- See memory requirements for the latest release (https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/jobs-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#jobs_memory_requirements)here (https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/jobs-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#jobs_memory_requirements).
+
+#### HTTP Compatibility Layer ( earlier HTTPS now changed to compatibility layer) v1.2.0
```
Comment (tanmoysen) [inline CHANGELOG.md:31]: HTTP Compatibility Layer v1.2.0

Btw, why is the compatibility layer v1.2.0? This is a new lib, right?

Comment (pavanmr94) [reply CHANGELOG.md:31]: this increments older library's version

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,12 +1,85 @@
...
+- The AWS IoT Jobs library has no dependencies on additional libraries other than the standard C library. It also doesn’t have any platform dependencies, such as threading or synchronization. It can be used with any MQTT library and any JSON library (see demos (https://github.com/aws/amazon-freertos/tree/master/demos/jobs_for_aws) with coreMQTT and coreJSON).
+- See memory requirements for the latest release (https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/jobs-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#jobs_memory_requirements)here (https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/jobs-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#jobs_memory_requirements).
+
+#### HTTP Compatibility Layer v1.2.0 (earlier HTTPS now changed to compatibility layer)
```
Comment (tanmoysen) [inline CHANGELOG.md:31]: just "HTTP Compatibility Layer v1.2.0"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,85 @@
...
+- See memory requirements for the latest release (https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/jobs-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#jobs_memory_requirements)here (https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/jobs-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#jobs_memory_requirements).
+
+#### HTTP Compatibility Layer v1.2.0 (earlier HTTPS now changed to compatibility layer) 
+
+- The HTTPS Compatibility Layer provides backwards compatibility from coreHTTP library to HTTP V1.x.x APIs. In addition, the HTTPS Compatibility Layer maintains the dependency of the network abstraction and linear containers from the HTTP V1.x.x library. The task pool dependency is removed in order to allow the user to allocate tasks on the stack rather than exclusively on the heap.
```
Comment (tanmoysen) [inline CHANGELOG.md:33]: "HTTP Compatibility Layer" (not HTTPS)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,12 +1,85 @@
...
+- Configuration settings using C preprocessor constants, for the HTTPS Compatibility Layer, are available in addition to the original configurations of the HTTPS V1.x.x library (https://docs.aws.amazon.com/freertos/latest/lib-ref/html2/https/https_config.html). They can be set with a #define in the config file (iot_config.h) or by using a compiler option such as -D in gcc. If a configuration setting is not defined, the HTTPS Compatibility Layer will use a "sensible" default value (unless otherwise noted). Because they are compile-time constants, this HTTPS Compatibility Layer must be rebuilt if a configuration setting is changed.
```
Comment (tanmoysen) [inline CHANGELOG.md:34]: "HTTP V1.x.x". Btw, is it HTTPS or HTTP. we should stay consistent throughout.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,12 +1,81 @@
...
+
+- Update logs and format specifiers to use standard C types.
+- Add dependence on stdbool.h
+- Introduce two new configuration macros for setting the timeout for transport reads and sends.
+
+#### coreJSON v3.0.0
+
+- Added an API function to iterate over a collection in a JSON document.
+
+#### Device Shadow v1.0.2
```
Comment (tanmoysen) [inline CHANGELOG.md:63]: AWS IoT Device Shadow

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,12 +1,81 @@
...
+
+- Update logs and format specifiers to use standard C types.
+
+#### PKCS11 v3.0.0
```
Comment (tanmoysen) [inline CHANGELOG.md:67]: is it corePKCS11 v3.0.0?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,12 +1,81 @@
...
+
+- Updated xFindObjectWithLabelAndClass to include a size parameter to allow passing in the size of the label, in order to avoid calling strlen in the library code.
+- Added CBMC memory proofs for all functions.
+- Removed threading_alt.h from corePKCS11.
+- Restructured third party folder in order to align with other core repositories. Folders located in “corePKCS11/3rdparty” are now in “corePKCS11/source/dependency/3rdparty”.
+- Updated logs and format specifiers to use standard C types.
+
+#### MQTT Compatibility layer v2.3.1
```
Comment (tanmoysen) [inline CHANGELOG.md:75]: MQTT Compatibility Layer ("L" capital)

File: CHANGELOG.md (hunk 6 above)
Comment (lundinc2) [reply CHANGELOG.md:67]: Yes

File: CHANGELOG.md (hunk 2 above)
Comment (pavanmr94) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (pavanmr94) [reply CHANGELOG.md]: taskpool dos not have a repository, should i link it to the path in this repo?

File: CHANGELOG.md (hunk 4 above)
Comment (pavanmr94) [reply CHANGELOG.md]: done, the compatibility layer is named "HTTP" while the previous library was "HTTPS"

File: CHANGELOG.md (hunk 5 above)
Comment (pavanmr94) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 6 above)
Comment (pavanmr94) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 7 above)
Comment (pavanmr94) [reply CHANGELOG.md]: done

**Review by lundinc2 — APPROVED** [review]

**Review by cobusve — APPROVED** [review]

**Pull request merged**